diffrax/integrate.py

```diff
@@ -599,7 +599,8 @@ def diffeqsolve(
 
     # Error checking
     if dt0 is not None:
-        error_if(lambda: (t1 - t0) * dt0 <= 0, "Must have (t1 - t0) * dt0 > 0")
+        msg = f"Must have (t1 ({t1, type(t1)}) - t0  ({t0, type(t0)})) * dt0  ({dt0, type(dt0)})> 0"
```

Comment:
So I find this a little hard to read. Can the `f"{}"` values be collected at the end of the error message?
When printing out a type, can we use plain English to denote what's going on, e.g. `Got value {t1} of type {type(t1)}.`